src/include/stir/Bin.h

```diff
@@ -79,6 +79,12 @@ class Bin
   
   //! accumulate voxel's contribution during forward projection 
   inline Bin&  operator+=(const float dx);
+  //! multiply bin values during normalisation -- apply()
```

Comment:
Now they don't make much sense, as I removed the bin-wise operations. But I will keep them for future reference.